INSTALL

```diff
@@ -16,6 +16,12 @@ the following command::
 
   conda install -c scitools iris
 
+If you wish to run any of the code examples
+(see http://scitools.org.uk/iris/docs/latest/examples/index.html) you will also
+need the Iris sample data. This can also be installed using Conda::
```

Comment:
The capitalisation of conda is inconsistent, everywhere else it is all lower case. You should use lower case here too.